src/main/index.js

```diff
@@ -229,6 +252,17 @@ app.on("ready", () => {
       const auth = await githubRepoInstance.getAuth();
       details.requestHeaders["Authorization"] = "bearer " + auth.token;
       callback({ requestHeaders: details.requestHeaders });
-    }
+    },
+   async () => {
+    const auth = await bitbucketRepoInstance.getAuth();
```

Comment:
filter is only looking for github's api, this will probably need more than just this.

Reply:
Implemented with the latest commit